pip_build.py

```diff
@@ -109,7 +129,18 @@ def install_whl(whl_fpath):
         action="store_true",
         help="Whether to install the generated wheel file.",
     )
+    parser.add_argument(
+        "--nightly",
+        action="store_true",
+        help="Whether to generate nightly wheel file.",
+    )
+    parser.add_argument(
+        "--rc",
```

Comment:
We don't generally do `rc` releases for cv or nlp, rather we just make a few "dev" releases for our own testing before we cut a real one. https://github.com/keras-team/keras-nlp/blob/master/RELEASE_PROCESS.md

Let's just cut this if it doesn't make sense for our process right now, or update the release process doc to reflect how to use this. No sense supporting an unused config option.

Reply:
Removed `rc`